src/components/cylc/table/Table.vue

```diff
@@ -240,40 +227,59 @@ export default {
         mdiChevronDown,
         mdiArrowDown
       },
-      sortBy: ['Cycle Point'],
+      sortBy: ['node.cyclePoint'],
       sortDesc: [localStorage.cyclePointsOrderDesc ? JSON.parse(localStorage.cyclePointsOrderDesc) : true],
       expanded: [],
       headers: [
         {
-          text: 'Task'
+          text: 'Task',
+          value: 'node.name',
+          sort: DEFAULT_COMPARATOR
         },
         {
           text: 'Jobs',
-          value: 'data-table-expand'
+          value: 'data-table-expand',
+          sortable: false
         },
         {
-          text: 'Cycle Point'
+          text: 'Cycle Point',
+          value: 'node.cyclePoint',
+          sort: (a, b) => DEFAULT_COMPARATOR(String(a ?? ''), String(b ?? ''))
         },
         {
-          text: 'Host'
+          text: 'Host',
+          value: 'latestJob.platform',
+          sort: (a, b) => DEFAULT_COMPARATOR(a ?? '', b ?? '')
         },
         {
-          text: 'Job System'
+          text: 'Job System',
+          value: 'latestJob.jobRunnerName',
+          sort: (a, b) => DEFAULT_COMPARATOR(a ?? '', b ?? '')
         },
         {
-          text: 'Job ID'
+          text: 'Job ID',
+          value: 'latestJob.jobId',
+          sort: (a, b) => parseInt(a ?? 0) - parseInt(b ?? 0)
         },
         {
-          text: 'T-submit'
+          text: 'T-submit',
+          value: 'latestJob.submittedTime',
+          sort: datetimeComparator
         },
         {
-          text: 'T-start'
+          text: 'T-start',
+          value: 'latestJob.startedTime',
+          sort: datetimeComparator
         },
         {
-          text: 'T-finish'
+          text: 'T-finish',
+          value: 'latestJob.finishedTime',
+          sort: datetimeComparator
         },
         {
-          text: 'dT-mean'
+          text: 'dT-mean',
+          value: 'meanElapsedTime',
+          sort: (a, b) => parseInt(a ?? 0) - parseInt(b ?? 0)
```

Comment:
Is this column actually used at all? I have never seen it contain anything when running workflows

Reply:
Unsure, @hjoliver?

Reply:
Checking ...

Reply:
... it should be used. It's supposed to show the mean elapsed run time.  GraphiQL shows the data is in the subscription.  Not surprisingly, actually, as it's used in computing task icon progress.  I'll attempt a separate fix PR.